### PR TITLE
Xang user workout exercise model

### PR DESCRIPTION
**Pull request by P-web-art**
… controller

# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change
Fixed spelling and added two lines back to the controller that i deleted

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

POSTMAN

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: server/api/controllers/UserControllers.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ function generateToken(user) {
 
 const register = (req, res) => {
   const { username, password } = req.body;
```
Comment (rusticlion) [inline server/api/controllers/UserControllers.js:17]: We were missing this destructuring and document creation code.

File: server/api/models/Exercise.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const ExerciseSchema = Schema({
+  name: {
+    type: String,
+    require: true
+  },
+  note: {
+    type: String
+  },
+  calendar: [
```
Comment (rusticlion) [inline server/api/models/Exercise.js:12]: Spelling corrected.

File: server/api/models/User.js [hunk 3]
```diff
@@ -14,7 +14,43 @@ const UserSchema = Schema({
...
+        type: Number,
+        required: true
+      }
+    }
+  ],
+  workouts: [
+    {
+      workout: { 
+        type: Schema.Types.ObjectId,
+        ref: "Workout"
```
Comment (rusticlion) [inline server/api/models/User.js:38]: Ref corrected

**Review by rusticlion — APPROVED** [review]
All typoes and syntax errors are repaired - looks ready for master!

**Review by emukupa — APPROVED** [review]
looks good

**Pull request merged**